griptape/drivers/prompt/cohere_prompt_driver.py

```diff
@@ -87,19 +99,145 @@ def _base_params(self, prompt_stack: PromptStack) -> dict:
             "temperature": self.temperature,
             "stop_sequences": self.tokenizer.stop_sequences,
             "max_tokens": self.max_tokens,
+            **({"tool_results": tool_results} if tool_results else {}),
+            **(
+                {"tools": self.__to_cohere_tools(prompt_stack.actions), "force_single_step": self.force_single_step}
+                if prompt_stack.actions and self.use_native_tools
+                else {}
+            ),
             **({"preamble": preamble} if preamble else {}),
         }
 
-    def __prompt_stack_content_message_content(self, content: BaseMessageContent) -> dict:
+    def __to_cohere_messages(self, messages: list[Message]) -> list[dict]:
+        cohere_messages = []
+
+        for message in messages:
+            new_message: dict = {"role": self.__to_cohere_role(message)}
```

Comment:
Maybe rename `new_message` to `cohere_message` for readability

Reply:
Done